packages/argon2/__test__/argon2.spec.ts

```diff
@@ -43,6 +43,16 @@ test('should be able to hash string', async (t) => {
   )
 })
 
+test('should be able to hashRaw string with a defined salt', async (t) => {
+  await t.notThrowsAsync(() => hash('whatever'))
```

Comment:
I'd either remove this assertion or providing a custom salt to the `hash` function

Reply:
yes totally that's a mistake i meant to write hashRaw

Reply:
I'm okay with this at now, so I'll merge it